docs/src/css/style.css

```diff
@@ -448,6 +454,11 @@ a.scroll-link:hover {
   }
 }
 
+.column.is-offset-2, .column.is-offset-2-tablet {
+  margin-left: 0% !important;
```

Comment:
using `!important` is not a good practice
you should try to manage specificity in other ways
you may want to read something about how CSS Specificity is calculated [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
@hugosolar I removed the !important please review